Backend/DataAccessLayer/Repositories/UserManagementRepository.cs

```diff
@@ -11,85 +11,56 @@ namespace DataAccessLayer.Repositories
 {
     public class UserManagementRepository
     {
-        public int CreateNewUser(User user)
+        public User CreateNewUser(User user, DatabaseContext _db)

```

Comment:
I think we should standardize that the dB context is always the first argument. That way it’s position is always known.